xlml/utils/xpk.py

```diff
@@ -62,7 +62,7 @@ def _run_workload(
       (
           "python3 xpk.py workload create"
           f" --cluster={cluster_name} --workload={workload_id}"
-          f" --command='{run_cmds}' --tpu-type={accelerator_type}"
+          f" --command='{run_cmds}' --device-type={accelerator_type}"
```

Comment:
This will be very tricky. I recall TPU will use `tpu-type`, and both GPU and CPU will use `device-type`. I wish this could be handled in xpk, so that we can only use `device-type` here.

Reply:
I think `device-type` can be used for tpu, gpu and cpu. https://github.com/google/xpk/blob/948a57a87cecfa61d668f5bf7b8c30b2c893df56/xpk.py#L4290

Reply:
+1 to what @NinaCai said. We plan to depreciate tpu-type one day but didn't want to break current users. device-type supports all accelerators.